openquake/smt/tests/comparison/data/compare_gmpe_inputs.toml

```diff
@@ -0,0 +1,113 @@
+### Input file for comparison of GMPEs using plotting functions in openquake.smt.comparison.compare_gmpes
+
+###For region parameter, we can use:
+#0 for global
+#1 for California
+#2 for Japan
+#3 for China
+#4 for Italy
+#5 for Turkey (locally = 3)
+#6 for Taiwan (locally = 0) 
+
+#Specify name of each run
+[run_names]
+name = ['Albania_1std','Albania_2std','Albania_3std']
+
+#Specify Vs30 for each run
+[vs30_values]
+vs30 = [800,800,600]
+
+#Specify Z1pt0 for each run
+[z1pt0_values]
+Z1 = [-999,-999,-999]
+
+#Specify z2pt5 for each run
+[z2pt5_values]
```

Comment:
Isn't this redundant? you could group Z1, Z25, Vs30 under a `site` group without having one group per each property (with the same name)